File: java/src/com/ten/hanabi/tests/RandomTest.java
package com.ten.hanabi.tests;

import java.util.*;

import com.ten.hanabi.core.*;

public class RandomTest {

	public static void main(String[] args) {
		HashSet<Color> cs = new HashSet<Color>();
		cs.add(Color.values()[1]);
		cs.add(Color.values()[1]);
		cs.remove(Color.values()[1]);
		System.out.println(cs.size());
	}

}

File: java/src/com/flecheck/hanabi/bga/tests/RandomTest.scala
package com.flecheck.hanabi.bga.tests

import com.flecheck.hanabi.bga.BGA
import com.ten.hanabi.ui.play.UIPlayManager

object RandomTest extends App{
  //val t = BGA.getGameById(27317631)
  //val t = BGA.getGameById(28015531)
  //val h = BGA.getGameById(28441973)
  
  //val h = BGA.getGameById(30709275)
  val h = BGA.getGameById(30820282)
  println(h)

  var upm: UIPlayManager = new UIPlayManager()
  upm.loadHanabi(h, 0)
}
